.travis.yml

```diff
@@ -3,17 +3,20 @@ sudo: false
 install:
   - mkdir bin
 
-matrix:
-    include:
-        - d: dmd-2.079.1
-        - d: dmd-2.078.3
-        - d: dmd-2.077.1
-        - d: ldc-1.8.0
-        - d: ldc-1.7.0
-        - d: ldc-1.6.0
+d:
+  - dmd
+  - dmd-2.080.0
+  - dmd-2.079.1
+  - dmd-2.078.3
+  - dmd-2.077.1
+  - ldc
```

Comment:
?

Reply:
same as `dmd` above, because otherwise new compilers only get tested when someone remembers to update this file

Reply:
They won't get tested with the new compilers until there's a push anyway. Much better to test nightlies and allow for CI failure in that particular case. At which point I'll add a cron job to the Travis config to run the build anyway every week.

Reply:
ok, removed

Reply:
main thing I wanted to do here was just the testLibrary testing anyway